trendy-fashion/src/Components/HomeKitchen.jsx

```diff
@@ -1,106 +0,0 @@
-import { Box,  Grid,    VStack } from "@chakra-ui/react";
-import React from "react";
-import "./Men.css"
-const HomeKitchen = () => {
-  return (
-    <Box w="100%" px={5} py={35}>
-      <Grid templateColumns="repeat(4,1fr)" gap={1} alignItems="start">
-        <VStack alignItems="flex-start" className="dropdown" >
-        <ul>
-        <li className='firstElement' >BED LINEN</li>
-        <li>Bedsheets</li>
-        <li>Bedding Sets</li>
-        <li>Blankets,Dohras & Quilts</li>
-        <li>Comforters</li>
-        <li>Bed Covers</li>
-        <li>Mattress Protectors</li>
-        <li>Quilt & Duvet Covers</li>
-        <li className='firstElement'>CUSHIONS & PILLOWS</li>
-        <li>Cushions</li>
-        <li>Pillows</li>
-        <li>Bed Wedges & Neck Pillows</li>
-        <li>Bolsters</li>
-       
-        <li>Cushion Covers </li>
-        <li>Pillow Covers</li>
-        
-        <li className='firstElement'>RUGS, CARPETS & MATS</li>
-       
-      </ul>
-
-        </VStack>
-        <VStack alignItems="start" className="dropdown" >
-        <ul>
-        <li className='firstElement' >CURTAIN & ACCESSORIES </li>
-        <li>Window Curtains</li>
-        <li>Door Curtains</li>
-        <li>Curtains</li>
-      
-        <li className='firstElement'>KITCHEN</li>
-        <li className='firstElement'>MENSWEAR</li>
-        <li>Cookware & Cultery</li>
-        <li>Bakeware</li>
-        <li>Kitchen Tools</li>
-        <li>Kitchen Aprons,Gloves & Towel</li>
-        <li>Kitchen Organisers</li>
-        
-        <li className='firstElement'>DINING</li>
-      <li>Serveware & Drinkware</li>
-      <li>Table Linen Sets</li>
-      <li>Table Covers & Runners</li>
-      <li>Table Napkins</li>
-      <li>Placemats & Coasters</li>
-      <li className='firstElement'>STATIONERY & ORGANISERS</li>
-      </ul>
-        </VStack>
-        <VStack alignItems="start" className="dropdown" >
-        
-        <ul>
-        <li className='firstElement' >HOME DECOR </li>
```

Comment:
Try to give some comments before starting each section to tell what this section is about.